docs/life_of_a_godwoken_transaction.md

```diff
@@ -222,14 +208,14 @@ A JSON representation for this data structure is:
 {
   "type": "SUDTTransfer",
   "value": {
-      "to": "0x4",
+      "to_address": "0x0200000014000000bb1d13450cfa630728d0390c99957c6948bf7d19",
       "amount": "0x190",
       "fee": "0x64"
   }
 }
 ```
 
-In this example, we are transferring 400 shannons(denoted by `to_id` 0x1) from account 0x2(denoted by `from_id`) to account 0x4(denoted by `to` in `SUDTTransfer`), we would also love to pay for 100 shannons as layer 2 transaction fees. The next 2 sections shall explain the details for layer 2 transaction.
+In this example, we are transferring 400 shannons(denoted by `to_id` 0x1) from account 0x2(denoted by `from_id`) to account 0x0200000014000000bb1d13450cfa630728d0390c99957c6948bf7d19(denoted by `to_address` in `SUDTTransfer`), we would also love to pay for 100 shannons as layer 2 transaction fees. The next 2 sections shall explain the details for layer 2 transaction.
```

Comment:
In this example, we are transferring 400 shannons (denoted by `to_id` 0x1) from account 0x2 (denoted by `from_id`) to account 0x0200000014000000bb1d13450cfa630728d0390c99957c6948bf7d19 (denoted by `to_address` in `SUDTTransfer`). In addition, we would like to pay 100 shannons as layer 2 transaction fees. The following two sections will discuss layer 2 transactions in more detail.